auto_max_lair.py

```diff
@@ -43,8 +43,19 @@
 LOG_NAME = f"{BOSS}_{datetime.now().strftime('%Y-%m-%d %H-%M-%S')}"
 
 
-def initialize(__) -> str:
+def initialize(ctrlr) -> str:
     """Placeholder. Immediately enter the join stage."""
+    # send a discord message that we're ready to rumble
+    ctrlr.send_discord_message(False, f"Starting a new full run for {ctrlr.boss}!",
+        embed_fields=ctrlr.get_stats_for_discord(), level="update"
+    )
+
+    # assume we're starting from the select controller menu, connect, then
+    # press home twice to return to the game
+    ctrlr.push_buttons(
+        (b'a', 2), (b'h', 2.0), (b'h', 2.0)
```

Comment:
```suggestion
        (b'a', 2), (b'h', 2.0), (b'h', 2.0), (b'b', 1.5), (b'b', 1.5)
```

- Modify the sequence so it works from the original starting location (in game facing the scientist).